### DISCUSSION TITLE
Maya 2025

### DISCUSSION THREAD

**Discussion by aldobenitez** [aronamao/Discord-Rich-Presence-For-Maya#7] [General]
Hi, would it be possible to create a version for 2025.2+ ?

Currently I get this error message:

import maya.app.general.executeDroppedPythonFile as myTempEDPF
myTempEDPF.executeDroppedPythonFile("C:/Users/Aldob/Downloads/m/installer.py", "")
del myTempEDPF
Error: ModuleNotFoundError: file C:\Users/Aldob/Downloads/m\installer.py line 5: No module named 'PySide2'

**Comment by aronamao** [comment]
I added 2025 support with release [1.3.0](https://github.com/aronamao/Discord-Rich-Presence-For-Maya/releases/tag/1.3.0)
Let me know if you're runnning into any issues 😄
